golang/pkg/datamodel/stateModelString.go

```diff
@@ -90,6 +91,79 @@ func ConvertStateToString(state int, languageCode LanguageCode) (stateString str
 		default:
 			stateString = fmt.Sprintf("Unbekannter Zustand %d", state)
 		}
+	} else if languageCode == 1 { // TURKISH
```

Comment:
Please keep 1 as the language code for english. turkish can get 2 :)

Reply:
Ok, I'll update now :)